cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -36,6 +36,8 @@ import (
 	"k8s.io/klog/v2"
 
 	"github.com/pkg/errors"
+	"go.etcd.io/etcd/api/v3/etcdserverpb"
+	"go.etcd.io/etcd/api/v3/v3rpc/rpctypes"
```

Comment:
just a non-blocking comment:

i find the etcd API package organization a bit odd.
i would have expected member types and common errors to have been abstract at the top level v3.

Reply:
Agreed :)